src/pages/signup/Third.tsx

```
@@ -0,0 +1,75 @@
+          <Button bigSize onClick={() => {}}>
+            중복 확인
+          </Button>
+        </EmailInput>
+        <SelectTagBox>
+          <Label>포지션</Label>
+          <TagBox>
+            {position.map((item) => (
+              <Tag>{item}</Tag>
+            ))}
+          </TagBox>
+        </SelectTagBox>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)